lib/assessment.rb

```diff
@@ -43,6 +43,8 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    File.read(filename) do |line|
+
+    end
```

Comment:
You've loaded the file correctly, but your setup with the block is slightly off. Files are Enumerable, so they can call `.each`, `.map`, etc. If you called one of those methods, the block you'd pass it would accept a single line as the argument. However, if you just give `File.open(filename)` a `do` block, it will be handed the file itself.